test/remote_mirrors.yml

```diff
@@ -2,6 +2,7 @@ mirrors:
   conda-forge:
     - https://conda.anaconda.org/conda-forge
     - https://repo.mamba.pm/conda-forge
+    - https://beta.mamba.pm/get/conda-forge
```

Comment:
repo.mamba.pm and beta.mamba.pm/get... point to the same thing. I am happy to keep this, just wondering what the motivation was :)

Reply:
Ya, it's the same domain... :-/
The intention was redundancy...